.gitignore

```diff
@@ -39,3 +39,6 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+# gitpod
+*.gitpod.yml
```

Comment:
`.gitpod.yml` is added on line 25 so you can remove one of these so it's not mentioned twice

Reply:
Thanks!